### PR TITLE
[Content] Korath Raider Update Edits

### PR DESCRIPTION
**Pull request by Saugia**
**Content**

## Summary

This PR addresses missing copyright, unnecessary lines, and any other errors or suggestions in the Korath Raider Update PR.

Here is a link to the original PR for reference: https://github.com/endless-sky/endless-sky/pull/6870

## Save File
This save file can be used to play through the new mission content: N/A

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/38
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/263

### CODE REVIEW

File: copyright [hunk 1]
```diff
@@ -1062,6 +1062,11 @@ Copyright: Benjamin Jackson (gods.benyamin@outlook.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license)
 
+Files:
+ image/effect/firestorm?ring*
+License: N/A (CC0 Public Domain)
```
Comment (quyykk) [inline copyright:1067]: That's not a valid license btw. Is it just public domain?

Comment (Saugia) [reply copyright:1067]: Nomadic created it but he labeled all his work under public domain

Comment (Saugia) [reply copyright:1067]: How would you change that?

**Comment by Anarchist2** [comment]
https://github.com/endless-sky/endless-sky/blob/3f2c4aaada9ca01c155df0f030f2c9d14931c05c/data/korath/korath%20ships.txt#L169-L170
Are we sure we want to be toying with the outfit space of ships here? Not sure if it's been brought up in the original PR or the main one, but it seems to me that adjusting the loadouts of the Raider would be much more convenient than having to deal with new and old Raiders having different stats or the balance headache of a stronger Raider.

https://github.com/endless-sky/endless-sky/blob/3f2c4aaada9ca01c155df0f030f2c9d14931c05c/data/korath/korath%20weapons.txt#L635-L636
What's a Firestorm torpedo? What's an Expeller gun? I know that these weapons are in the main Korath PR, but we don't want any hanging references like this if only the Firestorm missile makes it into the next release.

Would you mind also cleaning up the old Korath Exile ship variants to remove the unnecessary sprite and description definitions? We might as well while we're here.

File: copyright (hunk 1 above)
Comment (Amazinite) [reply copyright:1067]: image

**Comment by ravenshining** [comment]
> Are we sure we want to be toying with the outfit space of ships here? Not sure if it's been brought up in the original PR or the main one, but it seems to me that adjusting the loadouts of the Raider would be much more convenient than having to deal with new and old Raiders having different stats or the balance headache of a stronger Raider.

I just spent like, two days straight adjusting the loadouts of the Raider (and dredger) for the Stage I PR.  I knocked the outfit space back down a bit, to 777, in the process, but I could go further down to 768 without screwing things up *too* much.

> Would you mind also cleaning up the old Korath Exile ship variants to remove the unnecessary sprite and description definitions? We might as well while we're here.

I'm not sure what you mean by that, but it sounds like an unimportant task that would save me having to deal with a lot of merge conflicts if we did it in the Stage I PR.

**Comment by Zitchas** [comment]
> but it seems to me that adjusting the loadouts of the Raider would be much more convenient than having to deal with new and old Raiders having different stats or the balance headache of a stronger Raider.

Based on a quick search, there are two planetary defence jobs in syndicate space that use Raiders, but everything else appears to be Remnant missions. The only other way to encounter Raiders is as a rare spawn in a few Eastern Syndicate systems, the Ember Waste in general, or invading Korath Exile territory.
- The two syndicate jobs can be taken a look at fairly easily, but the system they're in spawn syndicate security naturally, so it could be just a matter of time untili the Korath are defeated there.
- Having a rare spawn in the Eastern systems be a bit tougher (and thus less likely to be disabled by pirates) is a benchmark change, in my opinion. As in, the Raiders should continue to be buffed until they are *not* routinely disabled by pirates. Having an unknown alien warship appear out of nowhere to pillage every human ship around *should* be an absolutely terrifying event for everyone. Until that is the case, we are failing.
- Ember Waste: Slightly more dangerous random spawn. Makes the EW areas slightly more dangerous. Sounds good to me.
- Exile territory: Having raiding into Exile territory itself be more dangerous is a 100% very good thing. Having people think long and hard about the wisdom of doing so is definitely good. The fact that people can no longer ditch their AM for more weaponry and/or regen is definitely a good thing.

On the flip side, any buffs to the Raider solves a range of lore holes:

1. The Korath are the most advanced aggressive and war-focused faction in the galaxy. They're basically the Wanderers or Heliarchs, except with a military/combat focus instead of terraforming and general post-scarcity-society-building. They *should* be more terrifying to go up against as either of those. After all, this is the race that *built* the Sestor, the Mereti, the nanite swarm so terrifying an *archon* has appointed itself guardian of them, and innumerable planet-killing weapons. Not to mention they are only one of three factions who figured out how to build JD. Their lore-based threat level is basically second only to the Drak. Which really doesn't jive with the ships they currently field, especially the Raider "mainstay" of their resource acquisition fleets. (My personal justification is that this is their "fallen" ship that is a cheap mass produced thing that doesn't require advanced resources or manufacturing capabilities, and isn't representative of their true capabilities; but that explanation rings like a copout to me)
2. The Syndicate has managed to acquire *seven* JD to turn over to the fleet, and this is framed as an accomplishment; and all that they've managed to achieve in the unknown length of time that the Korath have been raiding into human space. The fact that a mega-corporation with access to significant fleets and knows there's salvage and R&D potential to be had might focus significant resources on the endeavor, and still only manage to acquire such a small number of JD implies that either:

- a) the Raiders are a lot stronger and smarter than they actually are, or 
- b) the Syndicate is a lot dumber and less interested than it looks. 

- I prefer option A, as it it makes the Syndicate look less incompetent. They're already taking a few black eyes in the course of the story, so removing the embarrassment that is the inability to figure out camping the Korath for as many JD as the Syndicate could want is beneficial. The stronger the Raiders are, the more justified the Syndicate is in *not* sending fleets out to capture them (why waste fleets that are needed elsewhere for novelties?). 
3. Less need for swarms. The more often we can frame Korath battles as the player taking on a single powerful ship, and the less we have to throw them against a *fleet* of Raiders, the better. The bigger the battles are, the more likely the player is going to feel pressured into needing a fleet of their own; and I'd personally prefer to avoid that. 

**tldr:** I support all the buffs the Raiders currently get from this, and oppose any reduction thereof. I actually think they should get *more* buffs, but I'm satisfied with what they're currently getting.

**Comment by warp-core** [comment]
> 2. The Syndicate has managed to acquire *seven* JD to turn over to the fleet, and this is framed as an accomplishment; and all that they've managed to achieve in the unknown length of time that the Korath have been raiding into human space. The fact that a mega-corporation with access to significant fleets and knows there's salvage and R&D potential to be had might focus significant resources on the endeavor, and still only manage to acquire such a small number of JD implies that either:
> 
> - a) the Raiders are a lot stronger and smarter than they actually are, or 
> - b) the Syndicate is a lot dumber and less interested than it looks. 
> 
> - I prefer option A, as it it makes the Syndicate look less incompetent. They're already taking a few black eyes in the course of the story, so removing the embarrassment that is the inability to figure out camping the Korath for as many JD as the Syndicate could want is beneficial. The stronger the Raiders are, the more justified the Syndicate is in *not* sending fleets out to capture them (why waste fleets that are needed elsewhere for novelties?). 

https://github.com/endless-sky/endless-sky/blob/master/data/human/free%20worlds%20reconciliation.txt#L2391
Suggests they acquired more Jump Drives, just didn't get to keep them.

The Alphas have apparently provided the Unfettered Hai with on the order of 100 Jump Drives, these may have been originally acquired by the Syndicate, and they also had a not insignificant collection of their own.

File: copyright (hunk 1 above)
Comment (Saugia) [reply copyright:1067]: I found some examples of it used in older copyright additions and used those.

**Comment by EjoThims** [comment]
Descriptions is a non-issue, imo, expressly because player can't buy Korath outfits anyway.

Even if for some yet unseen reason they aren't appropriate to add in the relevant PR in the near future, they'll just be a clue for later things to find with the Korath.

Breadcrumbs for players to find and follow are good things.

**Comment by Zitchas** [comment]
@Anarchist2 While I agree that telling the player X exists when it doesn't would be less than ideal, I think it is perfectly acceptable to refer to things that may no longer exist, that simply don't exist, or that those who control it haven't revealed yet. Could be as simple as being a slip of the tongue by an overly enthusiastic engineer mistakenly revealing secrets.

For example, we consider repulsor engines as the canonical method of launching from gravity wells, but there's not a single repulsor engine to be seen.

That being said, while it does add slightly to the amount of changes in the main Korath PR to remove the line temporarily, it's an easy enough solution.

**Comment by Amazinite** [comment]
> Descriptions is a non-issue, imo, expressly because player can't buy Korath outfits anyway.

Captured outfits can still have their descriptions read. It's ships we don't need to worry about too much at the moment.

**Comment by Saugia** [comment]
I will do what raven suggested as it's an easy fix.

**Comment by EjoThims** [comment]
> > Descriptions is a non-issue, imo, expressly because player can't buy Korath outfits anyway.
> 
> 
> 
> Captured outfits can still have their descriptions read. It's ships we don't need to worry about too much at the moment.

I meant that since we can't buy the outfits directly, there's no problem in the implication that other outfits exist beyond our current ability to capture.

**Comment by Amazinite** [comment]
"Raider Mark XLII" feels like an odd name to me. There aren't any other ships in the game with such a naming scheme, and one could ask why it would be the case that this ship has its "Mark" number but the normal raider that we see doesn't. Is that the Mark I? The Mark XLI? We don't know, but the fact that only one of the ships lists its mark number just seems odd to me. It might be appropriate to rename it to something else. The original PR describe this as essentially a Marauder Raider. While Marauder is a specific term used for ships upgraded by pirates, we could perhaps think of some other adjective to use for this. "Korath Heavy Raider"? "Korath Assault Raider"? Or perhaps some different name entirely. "Korath Plunderer"? Just something other than "Mark XLII," which kind of sticks out like a sore thumb when compared to other ship names. (Adding a mark number to a name feels more fitting for a variant identifier as opposed to a ship's display name, which we actually do see for "Mark II" variants of Navy ships.)

**Comment by ravenshining** [comment]
> "Raider Mark XLII" feels like an odd name to me. 
>  "Korath Heavy Raider"? "Korath Assault Raider"? Or perhaps some different name entirely. "Korath Plunderer"?

Over on the Korath server, we've been discussing the idea of changing the naming convention for Korath ships entirely.  There's only so many aggressive-sounding verbs out there, and with the Coalition using the same naming convention, it's becoming difficult to find new names for new ships.  We'd like to move to a different scheme, something more unique to the Korath, and ideally that means renaming the Raider and the Raider Mark XLII.

The most popular option has been to use native Korath names for their ships - as if they were being named after famous Korath personalities or locations, much like RL Kuznetsov-, Kola-, Burke-, and Iowa- class aircraft carriers, frigates, cruisers, and battleships, respectively.  This would provide a connection to the Sestor, who use a combination Korath namelet and number.  And, it echoes the original naming scheme for Korath engines and reactors.

**Comment by Saugia** [comment]
For the sake of the current PR instead of renaming everything, I'll do a small change like "Korath Assault Raider" for now, and we can open a new one for all Korath Ships.

**Comment by ravenshining** [comment]
> For the sake of the current PR instead of renaming everything, I'll do a small change like "Korath Assault Raider" for now, and we can open a new one for all Korath Ships.

Right.  As far as the timing goes, I think it would be best to introduce the new naming scheme as part of Stage II, with the world-ship updates.  Stage I is essentially complete, and the less we need to edit/update that, the better, while Stage II will be doing some renaming *anyway*.

**Comment by Zitchas** [comment]
Yeah, I'd prefer to leave the name alone for now and change all the names together.
